app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/UserPhotosViewAdapterTest.kt

```diff
@@ -0,0 +1,80 @@
+package com.github.sdp_begreen.begreen.fragments
+
+import android.widget.LinearLayout
+import androidx.fragment.app.commit
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.assertThat
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.platform.app.InstrumentationRegistry
+import com.github.sdp_begreen.begreen.R
+import com.github.sdp_begreen.begreen.activities.MainActivity
+import com.github.sdp_begreen.begreen.databinding.FragmentUserPhotoBinding
+import com.github.sdp_begreen.begreen.models.ParcelableDate
+import com.github.sdp_begreen.begreen.models.PhotoMetadata
+import com.github.sdp_begreen.begreen.models.User
+import org.hamcrest.CoreMatchers
+import org.hamcrest.CoreMatchers.equalTo
+import org.hamcrest.MatcherAssert
+import org.junit.Rule
+import org.junit.Test
+import java.util.*
+
+class UserPhotosViewAdapterTest {
+    @get:Rule
+    val activityRule = ActivityScenarioRule(MainActivity::class.java)
+    private val photoList = listOf(
+        PhotoMetadata("1", "title", ParcelableDate(Date()), User("1", 812, "Alice", 3), "Gros vilain pas beau", "desc"),
+        PhotoMetadata("2", "title2", ParcelableDate(Date()), User("2", 81, "Bob", 4), "Gros vilain tout beau", "desc2")
+    )
+    private var userPhotoViewAdapter = UserPhotosViewAdapter(photoList, true)
+    private val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+
+    @Test
+    fun userViewAdapterGetItemCountWorksOnTrivialList() {
+        MatcherAssert.assertThat(userPhotoViewAdapter.getItemCount(), CoreMatchers.equalTo(2))
```

Comment:
This would be more readable with an import of `MatcherAssert` and `CoreMatchers` so that you don't need to repeat their class names every time